notebooks/book2/29/kf_parallel.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        if is_latexify_enabled():
</pre><p>Here also we don't need if...else, it's fine if notebook's figure don't have legend</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>